tests/PSRule.Tests/PSRule.Common.Tests.ps1

```diff
@@ -1125,6 +1125,119 @@ Describe 'Invoke-PSRule' -Tag 'Invoke-PSRule','Common' {
             $outVerbose[1] | Should -Be 'Rule verbose message 2';
         }
     }
+
+    Context 'Suppression output warnings' {
+        BeforeAll {
+            $testObject = @(
+                [PSCustomObject]@{
+                    Name = "TestObject1"
+                }
+                [PSCustomObject]@{
+                    Name = "TestObject2"
+                }
+            )
+        }
+
+        Context 'Detail' {
+            It 'From Parameter - Has warnings' {
+                $option = New-PSRuleOption -SuppressTargetName @{ FromFile1 = 'TestObject1'; FromFile2 = 'TestObject1'; } -SuppressedRuleWarning $True -OutputAs Detail;
+
+                $Null = $testObject | Invoke-PSRule -Path $ruleFilePath -Option $option -Name 'FromFile1', 'FromFile2' -WarningVariable outWarnings -WarningAction SilentlyContinue;
+    
+                $warningMessages = $outwarnings.ToArray();
+                $warningMessages.Length | Should -Be 2;
```

Comment:
@ArmaanMcleod Maybe filter the invariant culture warning first. We should probably add a option to disable this.

Reply:
@BernieWhite Ah thats probably whats causing one of the jobs to fail. I've tried setting the culture first in `BeforeAll` then resetting it in `AfterAll`. If that doesn't work I can try to filter the warnings first.